det3d/datasets/nuscenes/nusc_common.py

```diff
@@ -453,15 +453,15 @@ def _fill_trainval_infos(nusc, train_scenes, val_scenes, test=False, nsweeps=10,
                 current_cs_rec = nusc.get(
                     "calibrated_sensor", curr_sd_rec["calibrated_sensor_token"]
                 )
-                car_from_current = transform_matrix(
+                lidar_SE3_egovehicle = transform_matrix(
                     current_cs_rec["translation"],
                     Quaternion(current_cs_rec["rotation"]),
                     inverse=False,
                 )
 
                 tm = reduce(
                     np.dot,
-                    [ref_from_car, car_from_global, global_from_car, car_from_current],
+                    [egot1_SE3_lidart1, world_SE3_egot1, egovehicle_SE3_world, lidar_SE3_egovehicle],
```

Comment:
Hi Hemanth, thanks for the updates. If these are multiplied together sequentially, I would expect the source and destination indices to match. The original source did that, right?
```python
 [ref_from_car, car_from_global, global_from_car, car_from_current],
```

I'm also a fan of the Python in-fix operator `@` for matrix multiplication.
I would expect something like
```python
[lidart1_SE3_egot1, egot1_SE3_world, world_SE3_egot0, egot0_SE3_lidart0],
```
since our convention is that
```python
p_lidar = lidar_SE3_ego * p_ego
```
@bhavya01 do you have any insights here?

Reply:
Yes, the original source did that

Reply:
Addressed what we discussed [here](https://github.com/hchittanuru3/CenterPoint/pull/3/commits/33028fb14a9d161e38d8830b136aba2465f5ee2a)